src/main/java/seedu/address/logic/InternshipMessages.java

```diff
@@ -14,8 +14,10 @@ public class InternshipMessages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_INTERNSHIP_DISPLAYED_INDEX = "The internship index provided is invalid";
-    public static final String MESSAGE_INVALID_TASK_DISPLAYED_INDEX = "The task index provided is invalid!";
+    public static final String MESSAGE_INVALID_INTERNSHIP_DISPLAYED_INDEX = "Index has to be a positive number "
+            + "(1,2,3...) and one of the displayed internship indexes.";
+    public static final String MESSAGE_INVALID_TASK_DISPLAYED_INDEX = "Task index has to be a positive number "
+            + "(1,2,3...) and one of the displayed task indexes.";
```

Comment:
.. and one of the displayed task indexes of the internship."
add the last part ^